js/models/purchase/Order.js

```diff
@@ -23,8 +23,21 @@ export default class extends BaseModel {
     };
   }
 
+  addAddress(sAddr) {
+    // this will convert and set an address from the settings
```

Comment:
This allows an address from the settings to be converted and added to the order model in the proper format.